### ISSUE TITLE
Getting stuck at building tutorial

### ISSUE THREAD

**Issue by altschuler** [openstreetmap/iD#8315]
If you trace the building in the tutorial too accurately you will be told that "you had some trouble placing the points at the building corners" and to try again. I had to look into the code to figure out why, otherwise it's impossible to know why it's happening.

https://github.com/openstreetmap/iD/commit/0256252caf22096247e3bd4b4c91b1527b2b07ee presumably introduced the issue, and this is the check that "fails"
https://github.com/openstreetmap/iD/blob/a13f1f135b422b6969ea38751142f893dd66b34b/modules/ui/intro/building.js#L144

This tracing lets you through
![image](https://user-images.githubusercontent.com/956928/105634672-8c14a780-5e5f-11eb-8008-643d130c2045.png)

But this doesn't
![image](https://user-images.githubusercontent.com/956928/105634732-e0b82280-5e5f-11eb-943d-8dddbda7fb01.png)

Not sure the latter is even preferable, but it's difficult to know why it's failing :)

**Comment by maro-21** [comment]
We usually draw buildings by their ground contour, not by their roof. So the first one is correct, because what you draw is just a covered part.

**Comment by altschuler** [comment]
Right, that makes sense. Then I think it would be good to either be more explicit in the task description (it currently says something like "trace the footprint as accurately as possible" which for a novice like myself might not be clear) or give a hint in the retry text (eg "for this building you should use six points"). It's easy to get stuck there.

Would gladly do a PR, but not (yet) sure how the localization data needs to be updated etc.

**Comment by bhousel** [comment]
> or give a hint in the retry text (eg "for this building you should use six points").

It's not that you need to use X points, it's that the points need to make nearly right angles.  
Maybe for this step of the tutorial, I could just change it to use a different building that doesn't have any portico on the front.

**Comment by quincylvania** [comment]
> Maybe for this step of the tutorial, I could just change it to use a different building that doesn't have any portico on the front.

Sounds good. There've been issues here before (#6353).
